bokeh/util/callback_manager.py

```diff
@@ -17,6 +17,8 @@ def _callback_argspec(callback):
     elif ismethod(callback):
         # in this case the argspec will have 'self' in the args
         return getargspec(callback)
+    elif isinstance(callback, partial):
+        return getargspec(callback.func)
```

Comment:
the whole point of partial is that it changes the signature of `func` right?

It looks like `partial.__call__` may be implemented in native code? Then we hit the problem that native methods aren't introspectable: https://bugs.python.org/issue1748064

But maybe inspect.signature works ... python 3 only ? ... https://docs.python.org/3/library/inspect.html#inspect.signature  

I think what we might have to do is give up if the callback isn't a function or method, and `__call__` isn't a function or method either? would that screen out native calls?